autogen.sh

```diff
@@ -3,14 +3,24 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-export LC_ALL=C
 set -e
+
+assert_deps() {
+  for dep; do
+    if ! command -v $dep > /dev/null; then
+      echo "configuration failed, please install $dep first"
+      return 1
+    fi
+  done
+}
+
+assert_deps aclocal autoreconf libtool pkg-config
```

Comment:
@hebasto yep, it seems like `libtool` binary is not always present so it can't be used. It's present in Arch `libtool` but it's not provided by Debian

Without this check, it looks like this:

```
Can't exec "libtoolize": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 293.
autoreconf: error: libtoolize failed with exit status: 2
```

So maybe it would still look a bit cleaner